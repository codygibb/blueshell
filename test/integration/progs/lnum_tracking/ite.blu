# Make sure correct lnum is tracked no matter how deep
# into if/else if/else the error is.

if false {
	print "?"
} else if false {
	print "?"
} else if 5 {
	print "?"
} else {
	print "?"
}
