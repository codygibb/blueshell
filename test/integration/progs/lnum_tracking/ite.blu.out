error@8: Incorrect_type (if, int, bool)
